### PR TITLE
[Capacity] unstake benchmarking

### LINKED ISSUE #920: Benchmark for `unstake_capacity`
Write benchmark for unstaking capacity extrinsic

Acceptance criteria:

- [ ] Benchmark exists for `unstake_capacity` in pallets/capacity/src/benchmarking.rs
- [ ] Weights have been generated and checked into the branch by the Jenkins benchmark runner
- [ ] The `unstake_capacity` extrinsic uses the weights from WeightInfo

### PR DESCRIPTION
**Pull request by johnmela**
# Goal
The goal of this PR is to add benchmarking for the unstake extrinsic in the Capacity pallet 

Closes #920

### CODE REVIEW

File: pallets/capacity/src/lib.rs [hunk 1]
```diff
@@ -452,19 +452,19 @@ impl<T: Config> Pallet<T> {
 		unstaker: &T::AccountId,
 		amount: BalanceOf<T>,
 	) -> DispatchResult {
-		let mut staking_account_details =
+		let mut staking_account =
```
Comment (johnmela) [inline pallets/capacity/src/lib.rs:455]: Refactored `staking_account_details` -> `staking_account` for consistency in the pallet

File: pallets/capacity/src/benchmarking.rs [hunk 2]
```diff
@@ -59,7 +59,7 @@ benchmarks! {
 		let block_number = 4u32;
 
 		let mut staking_account = StakingAccountDetails::<T>::default();
-		staking_account.increase_by(10u32.into());
+		staking_account.increase_by(500u32.into());
```
Comment (johnmela) [inline pallets/capacity/src/benchmarking.rs:62]: Increase token amount to resolve an underflow error in the withdraw_unstaked benchmarking

**Review by shannonwells — APPROVED** [review]

File: runtime/frequency/src/lib.rs [hunk 3]
```diff
@@ -195,7 +195,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: spec_name!("frequency"),
 	impl_name: create_runtime_str!("frequency"),
 	authoring_version: 1,
-	spec_version: 10,
+	spec_version: 11,
```
Comment (enddynayn) [inline runtime/frequency/src/lib.rs:198]: I would not worry too much about this check failing.

**Review by enddynayn — APPROVED** [review]
💯 Nice!

**Pull request merged**